### ISSUE TITLE
[value=""] gets unquoted

### ISSUE THREAD

**Issue by arperry** [ben-eb/postcss-minify-selectors#19]
In 2.0.5, if value was empty, then canUnquote returned false:
```
    if (value) {
        value = value.replace(escapes, 'a').replace(/\\./g, 'a');
        return !(range.test(value) || /^(?:-?\d|--)/.test(value));
    }
    return false;
```

2.0.6 removes that check, so `[value=""]` gets converted to `[value=]`, which breaks the CSS.

**Comment by jable145** [comment]
I am having the same issue.

**Comment by ben-eb** [comment]
Resolved in `2.0.7`.
